lib-injection/copy-lib.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+cp -R monitoring-home/* "$1"
```

Comment:
I don't see anywhere that we configure/create the logs directory - do we need to do that here? Or in the lib injection controller?

Reply:
This is a good point. We don't currently execute any code in the application pod, we only:
1. Copy Tracer binaries into a shared location
2. Modify the application pod's environment variables

We will still have the `createLogPath.sh` so if/when we intercept the application container's startup, we can invoke the `createLogPath.sh` that we've placed on-disk. For now, we may not have log files and I'll write this down as a limitation

Reply:
> For now, we may not have log files and I'll write this down as a limitation

Another potential option would be to write the logs to a shared location too (e.g. "next" to the binaries) and inject the `DD_TRACE_LOG_DIRECTORY`?